README.md

```diff
@@ -2,7 +2,7 @@
 
 React is a JavaScript library for building user interfaces.
 
-* **Declarative:** React makes it painless to create interactive UIs. Design simple views for each state in your application, and React will efficiently update and render just the right components when your data changes. Declarative views make your code more predictable, simpler to understand, and easier to debug.
+* **Declarative:** Create interactive UIs effortlessly using React. Design simple views for each state in your application, and React will efficiently update and render just the right components when your data changes. Declarative views make your code more predictable, simpler to understand, and easier to debug.
```

Comment:
The biggest issue I have with this is that you've moved **the what** to the end of the sentence.

Apart from that I like the word **painless** more because I think, doing anything meaningful always requires some kind of **effort** but can be **painless** at the same time, what React is trying to achieve.

Reply:
Yes, I agree to this as well.

I am a newbie in open source, do you have any suggestions for me, as to where I can contribute. Any sort of help or guideline.

Reply:
If you know some piece of technology well, I would suggest simply helping others by answering their questions on forums, under videos etc.
Helping always feels good :)

Reply:
Can I consider **discord** servers related to coding, **twitter** or even **youtube** tutorials  as one of these so called forums?
Or do you mean something else?